e2e/tests/init_test.go

```diff
@@ -219,14 +220,31 @@ func (s *suite) TheUserStakesAWithUpoktFromTheAccount(actorType string, amount i
 
 func (s *suite) TheUserStakesAWithUpoktForServiceFromTheAccount(actorType string, amount int64, serviceId, accName string) {
 	// Create a temporary config file
-	configPathPattern := fmt.Sprintf("%s_stake_config_*.yaml", accName)
+	configPathPattern := fmt.Sprintf("%s_stake_config.yaml", accName)
 	configFile, err := os.CreateTemp("", configPathPattern)
 	require.NoError(s, err, "error creating config file in %q", path.Join(os.TempDir(), configPathPattern))
 
-	configContent := fmt.Sprintf("stake_amount: %d upokt\nservice_ids:\n  - %s", amount, serviceId)
+	// Prepare the config content based on the actor type
+	var configContent string
+	switch actorType {
+	case "application":
+		configContent = fmt.Sprintf("stake_amount: %d upokt\nservice_ids:\n  - %s", amount, serviceId)
+	case "supplier":
+		configContent = fmt.Sprintf(`stake_amount: %dupokt
```

Comment:
Maybe you could use `NormalizeYAMLIndentation` from `testutil/yaml` and keep code indentation